### ISSUE TITLE
<html> in railsport doesn't include the dir="" attribute

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#1930]
**[Submitted to the original trac issue database at 5.32pm, Sunday, 7th June 2009]**

This:
```
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
```
Should be this:
```
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
```

And for rtl languages it should be rtl. Probably best to include LTR/RTL at the top of the language file.

**Comment by tomhughes** [comment]
**Author: smsm1**
**[Added to the original trac issue at 6.44pm, Sunday, 7th June 2009]**

(In [15753]) include the correct language and direction in the html element. fixes #1930. fixes #1929

**Comment by tomhughes** [comment]
**Author: StefanB**
**[Added to the original trac issue at 5.39am, Friday, 3rd July 2009]**

for some reason the <% t'html.dir' %> renders empty in any language

**Comment by tomhughes** [comment]
**Author: stefanb**
**[Added to the original trac issue at 5.49am, Friday, 3rd July 2009]**

(In [16287]) added missing =, fixes #1930
